src/main/java/com/dzaro/DateBaseCon.java

```diff
@@ -37,7 +37,7 @@ public void deleteAllTable() {
     public void createAllTable() {
```

Comment:
Consider renaming this function:
```suggestion
    public void createTables() {
```